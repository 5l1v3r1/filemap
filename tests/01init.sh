#!/bin/sh

rm -Rf /tmp/fmtest
fm -v init
find /tmp/fmtest | sort

