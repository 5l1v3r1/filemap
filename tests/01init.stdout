/tmp/fmtest
/tmp/fmtest/n0.3
/tmp/fmtest/n0.3/sys
/tmp/fmtest/n0.3/sys/fm
/tmp/fmtest/n0.3/reduce
/tmp/fmtest/n0.3/jobs
/tmp/fmtest/n2.3
/tmp/fmtest/n2.3/sys
/tmp/fmtest/n2.3/sys/fm
/tmp/fmtest/n2.3/reduce
/tmp/fmtest/n2.3/jobs
/tmp/fmtest/n0.2
/tmp/fmtest/n0.2/sys
/tmp/fmtest/n0.2/sys/fm
/tmp/fmtest/n0.2/reduce
/tmp/fmtest/n0.2/jobs
/tmp/fmtest/n1.2
/tmp/fmtest/n1.2/sys
/tmp/fmtest/n1.2/sys/fm
/tmp/fmtest/n1.2/reduce
/tmp/fmtest/n1.2/jobs
/tmp/fmtest/n0.1
/tmp/fmtest/n0.1/sys
/tmp/fmtest/n0.1/sys/fm
/tmp/fmtest/n0.1/reduce
/tmp/fmtest/n0.1/jobs
/tmp/fmtest/n2.1
/tmp/fmtest/n2.1/sys
/tmp/fmtest/n2.1/sys/fm
/tmp/fmtest/n2.1/reduce
/tmp/fmtest/n2.1/jobs
/tmp/fmtest/n1.1
/tmp/fmtest/n1.1/sys
/tmp/fmtest/n1.1/sys/fm
/tmp/fmtest/n1.1/reduce
/tmp/fmtest/n1.1/jobs
/tmp/fmtest/n1.3
/tmp/fmtest/n1.3/sys
/tmp/fmtest/n1.3/sys/fm
/tmp/fmtest/n1.3/reduce
/tmp/fmtest/n1.3/jobs
/tmp/fmtest/n2.2
/tmp/fmtest/n2.2/sys
/tmp/fmtest/n2.2/sys/fm
/tmp/fmtest/n2.2/reduce
/tmp/fmtest/n2.2/jobs
